spec/menu_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'menu'
+
+describe Menu do
+
```

Comment:
Maybe could mock the menu so that you can be clear that the menu works with other menu items